panacea/query_client.go

```diff
@@ -84,6 +86,7 @@ func newQueryClient(ctx context.Context, config *config.Config, info *TrustedBlo
 	store := dbs.New(db, chainID)
 
 	var lc *light.Client
+	logger := light.Logger(tmlog.NewTMLogger(os.Stdout))
```

Comment:
How about using the `newSyncWriter()` as [tendermint does](https://github.com/tendermint/tendermint/blob/2ae117ef6288f810f3268323ccaba6cbc5e937ce/cmd/tendermint/commands/light.go#L105)?
It is concurrent safe writer and it would be good to us as well since we may have several go-routines for each event.

Reply:
Thanks, it's good to our go-routines event. I fixed it in [f7b488e](https://github.com/medibloc/panacea-doracle/pull/46/commits/f7b488e1dd94f4dfc16eac4e9b5036b9ba09d41c).